src/main/java/reposense/parser/ArgsParser.java

```diff
@@ -110,12 +114,16 @@ public static CliArguments parse(String[] args) throws ParseException {
             Path outputFolderPath = results.get("output");
             Optional<Date> sinceDate = results.get("since");
             Optional<Date> untilDate = results.get("until");
-            List<String> formats = results.get("formats");
-            List<String> locations = results.get("repos");
+            List<Format> formats = Format.convertStringsToFormats(results.get("formats"));
+            List<String> locationStrings = results.get("repos");
```

Comment:
Any reason not to use this convention? Its pretty neat.

`List<Location> locations = Location.convertStringsToLocations(results.get("repos"));`

Reply:
Good point